blog/2023/05-14-android-v250/index.mdx

```diff
@@ -0,0 +1,80 @@
+---
+title: 'Jellyfin Android v2.5.0 - Bitrate selection, player improvements and quality-of-life tweaks'
+description: 'Enjoy a more polished Jellyfin experience on your phone'
+authors: Maxr1998
+date: 2023-05-14
+slug: android-v2.5.0
+---
+
+It has been a long while since the last update, but after an extended beta testing period with plenty of bug reports (thanks, by the way!) and subsequent fixes, we're excited to finally announce a new major update to Jellyfin Android.
+
+Version 2.5.0 again focused on improvements to the integrated player, and brings various new features, many bug fixes and a lot of technical maintenance and rewrites.
```

Comment:
```suggestion
Version 2.5.0 again focuses on improvements to the integrated player, and brings various new features, many bug fixes and a lot of technical maintenance and rewrites.
```

Not sure about the "again" part, maybe write it as "just like the previous version" or something.

Reply:
I'd like to keep the again, 2.4.0 was similar after all.